main.go

```diff
@@ -545,6 +545,7 @@ func (e *Exporter) parseStats(ch chan<- prometheus.Metric, stats map[net.Addr]me
 		"outofmemory":       e.itemsOutofmemory,
 		"reclaimed":         e.itemsReclaimed,
 		"tailrepairs":       e.itemsTailrepairs,
+		"mem_requested":     e.slabsMemRequested,
```

Comment:
What do you think, should we also rename the metric name to follow the upstream project?

Reply:
Maybe, I think it's still technically slab memory. For now I was just thinking about avoiding a breaking change since the old version is still popular.

Reply:
The change is listed as "internal cleanup" and just moves the value from one status command to another. So I think we should just keep this the same.